tools/RAiDER/statsPlot.py

```diff
@@ -620,17 +628,17 @@ def __call__(self, gridarr, plottype, workdir='./', drawgridlines=False, colorba
         '''
             Visualize a suite of statistics w.r.t. stations. Pass either a list of points or a gridded array as the first argument. Alternatively, you may superimpose your gridded array with a supplementary list of points by passing the latter through the stationsongrids argument.
         '''
-
+        import cartopy.crs as ccrs
+        import cartopy.feature as cfeature
+        import cartopy.io.img_tiles as cimgt
         import matplotlib as mpl
+        import matplotlib.ticker as mticker
+        from cartopy.mpl.ticker import LatitudeFormatter, LongitudeFormatter
+        from mpl_toolkits.axes_grid1 import make_axes_locatable
+        from pandas.plotting import register_matplotlib_converters
```

Comment:
@sssangha can you update the conda requirements file for any dependencies needed by the stats class?

Reply:
Also I'm nervous about the cartopy dependency; do you know if there are any conflicts with GDAL?

Reply:
@jlmaurer in my experience no, I haven't experienced any conflicts with GDAL.